packages/client/src/components/formConfig/formTools/DefaultFieldTools.tsx

```diff
@@ -124,7 +124,11 @@ export function DefaultFieldTools({
           {intl.formatMessage(messages.contentKey)}
           <StyledTooltip />
         </Subtitle>
-        <Body>{contentKey}</Body>
+        <Body>
+          {contentKey.map((content, index) => (
+            <div key={index}>{content}</div>
```

Comment:
```suggestion
            <span key={index}>{content}</span>
```
Span is probably a better choice here

Reply:
Agree